wp-content/plugins/core/src/Panels/Types/Wysiwyg.php

```diff
@@ -12,6 +12,10 @@ class Wysiwyg extends Panel_Type_Config {
 	const FIELD_DESCRIPTION    = 'description';
 	const FIELD_COLUMNS        = 'columns';
 	const FIELD_COLUMN_CONTENT = 'column_content';
+	const FIELD_LAYOUT         = 'layout';
+
+	const OPTION_LAYOUT_LEFT   = 'option_wysiwyg_layout_left';
```

Comment:
See here: https://github.com/moderntribe/square-one/pull/355#discussion_r313149269 and how that flows into the wrapper layout class ;)

Reply:
Updated